source/includes/azure/_security_rules.md

```diff
@@ -0,0 +1,82 @@
+### Security Rules
+
+A security rule in Azure is a filter which controls both inbound and outbound traffic on a network security group attacted to the ressource that receives the traffic.
+
+<!-------------------- LIST SECURITY RULES -------------------->
+
+#### List security rules
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/azure/example/securityrules?security_group_id=:securitygroupid"
+
+# Example:
+```
+```json
+{
+  "data": [
+    {
+        "id": "/subscriptions/subscription/resourceGroups/example-system-azure-example/providers/Microsoft.Network/networksecuritygroups/sample-network-security-group/defaultSecurityRules/SampleRuleInBound",
+        "name": "SampleRule",
+        "priority": 120,
+        "direction": "Inbound",
+        "access": "Allow",
+        "protocol": "*",
+        "sourcePortRanges": [],
+        "sourceAddressPrefixes": [],
+        "destinationPortRanges": [
+            "8080"
+        ],
+        "destinationAddressPrefixes": [
+            "10.0.0.0/24"
+        ]
+    },
+    {
+      "id": "/subscriptions/subscription/resourceGroups/example-system-azure-example/providers/Microsoft.Network/networksecuritygroups/sample-network-security-group/defaultSecurityRules/SampleRuleInBound2",
+      "name": "SampleOutbound",
+      "priority": 110,
+      "direction": "Outbound",
+      "access": "Deny",
+      "protocol": "Udp",
+      "sourcePortRanges": [
+        "1024"
+      ],
+      "sourceAddressPrefixes": [
+        "0.0.0.1",
+        "10.0.0.0/32"
+      ],
+      "destinationPortRanges": [
+        "8080"
+      ],
+      "destinationAddressPrefixes": [
+        "10.0.0.0/16"
+      ],
+    }
+  ],
+  "metadata": {
+    "recordCount": 2
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/securityrules?security_group_id=<a href="#azure-network-security-groups">:security_group_id</a></code>
+
+Retrieve a list of all security rules in an [network security group](#azure-network-security-groups)
+
+Query parameters | &nbsp;
+---------- | -----
+`direction`<br/>*string* | Filter on the list of security rules. Either `Inbound` or `Outbound`. 
```

Comment:
Don't you need the `security_group_id`?  

As an aside, it's a bit odd to have force having a query param also on a list, though it is tricky because rules are always children of security groups. Maybe we can ask on slack what should be the default behaviour with no filter

Reply:
This isn't a required query parameter. Not passing it will fetch the entire list, hence why I wrote "filter on the list"

Reply:
Added a mention about it.